packages/@react-spectrum/s2/src/Table.tsx

```diff
@@ -11,85 +11,1089 @@
+
+// TODO: placeholder, just copied over from v3. Request filed in airtable for the actual nubbin
+function Nubbin() {
   return (
-    <AriaRow id={id} {...otherProps}>
-      {allowsDragging && (
-        <Cell>
-          <Button slot="drag">≡</Button>
-        </Cell>
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+      <g fill="var(--nubbin-color)" stroke="var(--nubbin-color)" strokeWidth="2">
+        <circle cx="8" cy="8" r="8" stroke="none" />
+        <circle cx="8" cy="8" r="7" fill="none" />
+      </g>
+      <path d="M-2106-7380.263v5l2.5-2.551Z" transform="translate(2116 7385.763)" fill="#fff" stroke="#fff" strokeLinejoin="round" strokeWidth="2" />
+      <path d="M-2106-7380.263v5l2.5-2.551Z" transform="translate(-2100 -7369.763) rotate(180)" fill="#fff" stroke="#fff" strokeLinejoin="round" strokeWidth="2" />
+    </svg>
+  );
+}
+
+
+function ResizableColumnContents(props: ResizableColumnContentProps) {
+  let {allowsSorting, sortDirection, sort, startResize, children, isHovered, align} = props;
+  let {setIsInResizeMode, isInResizeMode} = useContext(InternalTableContext);
+  let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-spectrum/s2');
+  const onMenuSelect = (key) => {
+    switch (key) {
+      case 'sort-asc':
+        sort('ascending');
+        break;
+      case 'sort-desc':
+        sort('descending');
+        break;
+      case 'resize':
+        setIsInResizeMode?.(true);
+        // TODO: Need to wait for the menu to fully transition out, otherwise mouse movements may hover the menu items and steal focus from the resizer,
+        // causing resizing to end prematurely due to blur. Open to ideas for how to handle this since it affects RAC. Ideally we'd freeze
+        // focus from being able to move if we are in resizing mode except if the user clicks away or moves focus with something other than hover
+        setTimeout(() => startResize(), 200);
+        break;
+    }
+  };
+
+  let items = useMemo(() => {
+    let options = [
+      {
+        label: stringFormatter.format('table.resizeColumn'),
+        id: 'resize'
+      }
+    ];
+    if (allowsSorting) {
+      options = [
+        {
+          label: stringFormatter.format('table.sortAscending'),
+          id: 'sort-asc'
+        },
+        {
+          label: stringFormatter.format('table.sortDescending'),
+          id: 'sort-desc'
+        },
+        ...options
+      ];
+    }
+    return options;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [allowsSorting]);
+
+
+  let buttonAlignment = 'start';
+  let menuAlign = 'start' as 'start' | 'end';
+  // TODO: align center is quite strange, copied this from S1 but really there isn't a good place to put the menu when the column text is centered
+  if (align === 'center') {
+    buttonAlignment = 'center';
+  } else if (align === 'end') {
+    buttonAlignment = 'end';
+    menuAlign = 'end';
+  }
```

Comment:
<img width="583" alt="image" src="https://github.com/user-attachments/assets/36130ebb-dc74-42ad-9dc5-536d6260fb41">
wonder if there should be an align center option for menu